src/platforms/node/guides/serverless-cloud/index.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Serverless Cloud
+redirect_from:
+  - /clients/node/integrations/serverless-cloud/
+  - /platforms/node/serverless-cloud/
```

Comment:
Since this is a new page, no redirect is required. They're used to ensure users can still get to a page when it's been moved. 
```suggestion
```